src/index.html

```diff
@@ -1,13 +1,48 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link
+     href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+     rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <div class="product-card__container">
+        <div class="product-card__image">
+          <img
+           src="images/imac.jpeg"
+           class="product-card__image"
+           alt="imac"
+          >
+        </div>
+        <p class="product-card__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
+        <p class="product-card__number">Код товара: 195434</p>
+        <div class="product-card__rating">
+          <div class="product-card__stars">
```

Comment:
stars should have a `--n` modifier, which defines how many active stars there gonna be